.github/workflows/release.yml

```diff
@@ -11,9 +11,13 @@ jobs:
       matrix:
         target:
           - os: linux
-            arch: "amd64"
+            arch: "amd64-gnu"
```

Comment:
This is breaking because it's what defines the artifact filename.

I have two propositions, you can decide:
* We upload the `sshs-linux-amd64-gnu` and we upload it a second time named `sshs-linux-amd64`
* We keep `sshs-linux-amd64` without the `-gnu`

What do you think is best?

Reply:
I think we can keep `sshs-linux-amd64` without the `-gnu`